test/integration.test.ts

```diff
@@ -177,27 +177,24 @@ it('merges when receiving status event', async () => {
     config
   })
 
-  const graphql = jest.fn(async (query, variables) => {
-    if (variables.refQualifiedName) {
-      return {
-        repository: {
-          ref: {
-            associatedPullRequests: {
-              nodes: [{
-                number: 1,
-                repository: {
-                  name: 'probot-auto-merge',
-                  owner: {
-                    login: 'bobvanderlinden'
-                  }
+  const graphql = jest.fn(async (query, variables) => github.graphql(query, variables)).mockResolvedValueOnce({
```

Comment:
```suggestion
  const graphql = jest.fn(async (query, variables) => github.graphql(query, variables))
    .mockResolvedValueOnce({
```
I'm sorry for the lack of linting rules. I think this makes the alignment more consistent.